test_helpers.rb

```diff
@@ -708,11 +708,12 @@ def sim_test(filename, options = {})
       cbor_target_path = File.join(files_dir, File.basename(cbor_ori_path))
 
       FileUtils.cp(cbor_ori_path, cbor_target_path)
-    when 'python_plugin.osm'
+    when 'python_plugin.osm', 'python_plugin_search_paths'
       # We need to manually copy the supporting schedule into
       # the testruns folder for the simulation to be able to find it
+      program_file_name = "#{File.basename(filename, File.extname(filename))}_program.py"
       plugin_ori_path = File.join(File.dirname(__FILE__),
-                                  'model/simulationtests/python_plugin_program.py')
+                                  'model/simulationtests', program_file_name)
```

Comment:
When running the OSM version, copy the program.py file, which is computed as "#{stem}_program.py

python_plugin.osm -> python_plugin_program.py
python_plugin_search_paths.osm -> python_plugin_search_paths_program.py